extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx

```diff
@@ -26,7 +26,7 @@ const SR_SOPCLASSHANDLERID =
  * @param {*} param0
  */
 function TrackedMeasurementsContextProvider(
-  { servicesManager, commandsManager, extensionManager }, // Bound by consumer
+  { servicesManager, commandsManager, extensionManager, appConfig }, // Bound by consumer
```

Comment:
You don't need to pass the appConfig as props in most of the changed files. Take a look at how we use `useAppConfig` hook

Reply:
Changed as requested